tensorflow_lite_support/python/task/vision/proto/class_pb2.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Move to `tensorflow_lite_support/python/task/core/proto`. Please follow the same structure as the proto folder structure.